CHANGELOG/1.3-CHANGELOG.md

```diff
@@ -0,0 +1,232 @@
+in a future version.
+
+This feature does not currently have a feature name defined.
+
+This feature's status is **Extended**, meaning that it is _optional_ for
+implementations to support.
+
+As there is no feature name or conformance testing available for this feature
+yet, please see your implementation's documentation to see if it is supported.
+
+Relevant PRs:
+- Implementing CORS Filter for `HTTPRoute` ([#3637](https://github.com/kubernetes-sigs/gateway-api/pull/3637), [@robscott](https://github.com/robscott))
+- Change `HTTPRouteFilter.CORS.AllowCredentials` to expect a boolean and not a string ([#3656](https://github.com/kubernetes-sigs/gateway-api/pull/3656), [@EyalPazz](https://github.com/EyalPazz))
+- Add CORS to `HTTPRouteFilterType` ([#3668](https://github.com/kubernetes-sigs/gateway-api/pull/3668), [@EyalPazz](https://github.com/EyalPazz))
+
+### ListenerSets (Standard Mechanism to Merge Gateways)
+[GEP-1713](https://gateway-api.sigs.k8s.io/geps/gep-1713/) defines a new mechanism to merge listeners into a single
+Gateway, and in this release, this change has moved to Experimental.
+
+Please see the GEP reference document or the API reference for the details.
+
+This feature has graduated to Experimental and should now be used for testing
+and verification purposes only. Experimental features may be changed or removed
+in a future version.
+
+This feature does not currently have a feature name defined.
+
+This feature's status is **Extended**, meaning that it is _optional_ for
+implementations to support.
+
+As there is no feature name or conformance testing available for this feature
+yet, please see your implementation's documentation to see if it is supported.
+
+Relevant PRs:
+- Clarified what it means for Gateway Listeners to be distinct ([#3477](https://github.com/kubernetes-sigs/gateway-api/pull/3477), [@youngnick](https://github.com/youngnick))
+- GEP-1713: Standard Mechanism to Merge Multiple Gateways ([#3213](https://github.com/kubernetes-sigs/gateway-api/pull/3213)), [@dprotaso](https://github.com/dprotaso))
+- Update GEP-1713 - Support attaching `ListenerSet`s across namespaces ([#3632](https://github.com/kubernetes-sigs/gateway-api/pull/3632), [@dprotaso](https://github.com/dprotaso))
+- GEP-1713: Standard Mechanism to Merge Multiple Gateways - move GEP Link to Experimental ([#3664](https://github.com/kubernetes-sigs/gateway-api/pull/3664)), [@gcs278](https://github.com/gcs278))
+- Refactor codegen scripts to make it easier to generate two clients ([#3589](https://github.com/kubernetes-sigs/gateway-api/pull/3589),[@dprotaso](https://github.com/dprotaso))
+- Add ListenerSet GEP-1713 to the website ([#3587](https://github.com/kubernetes-sigs/gateway-api/pull/3587),[@dprotaso](https://github.com/dprotaso))
+- Introduces `ListenerSet` API & Generate Clients (in the group gateway.networking.k8s-x.io) ([#3588](https://github.com/kubernetes-sigs/gateway-api/pull/3588), [@dprotaso](https://github.com/dprotaso))
```

Comment:
This section will need an update after #3682 merges - I think we'll need to add something about how because it's Experimental, the objects are currently called XListenerSet etc.

Reply:
#3682 has merged, so we'll need to put an explanation in about experimental objects being called XObject.